webapp/tests/karma/ts/services/db.service.spec.ts

```diff
@@ -21,6 +21,7 @@ describe('Db Service', () => {
   let pouchResponse;
   let runOutsideAngular;
   let runInsideAngular;
+  const sandbox = sinon.createSandbox();
```

Comment:
Can you elaborate on this change? I see that a [sandbox has a benefit of easier cleanup](https://sinonjs.org/releases/latest/sandbox/), but I don't see any cleanup-like code changing below. Were we missing cleanup previously and now this handles it better for us?

Reply:
I was reverting this change right just a few moments ago. I initially thought it was solving cleanup issues but I was wrong, it's working exactly the same way with and without this explicit sandbox.